tasks/extract_filepaths_for_granule/schemas/input.json

```diff
@@ -10,6 +10,10 @@
           {
             "type": "object",
             "properties": {
+              "recoverybucketoverride": {
+                "description": "The user specified recovery S3 bucket to store recovered files",
+                "type": ["string", "null"]
+              },
```

Comment:
Remember to include in changelog.

Reply:
did update the changelog

Reply:
Whoops. My eyes glanced over it as a folder since the other changes were under it.